docker-entrypoint.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env ash
+set -euxo pipefail
+
+function _prepare() {
+	# Ensure that the bot can write to the storage
+	chown --recursive free-stuff-matrixbot:free-stuff-matrixbot "$( dirname -- "${STORAGE_FILE}" )"
+}
+
+function _run() {
+	# Run the bot not as root
+	su-exec free-stuff-matrixbot:free-stuff-matrixbot /usr/local/bin/python free-stuff-matrixbot.py
+}
+
+function _main() {
+	case ${1} in
+		run)
+			_prepare
+			_run
+		;;
+		*)
+			echo "Sub command required: run"
```

Comment:
We _could_ put an `exit 1` after this to make sure we exit with an error - this would allow only restarting this on errors etc.  
Your call if we want to do that though. :D

Reply:
Good question. If we exit with an error, than the container would be restarted every time in certain environments, but yes, I think that's a problem for the environment than and good environments have back off on errors (like k8s with `CrashLoopBackOff`)